app/lib/get-all-project-preferences.js

```diff
@@ -0,0 +1,20 @@
+function getAllProjectPreferences(user, _page = 1, _accumulator = []) {
+  return user.get('project_preferences', {
```

Comment:
This call should specify `?include=project` which will fix the N-request issue on https://github.com/zooniverse/Panoptes-Front-End/pull/2813/files#diff-7ba34496843ed4dbf5a57e4c5dc0dd3aR6